com.unity.render-pipelines.high-definition/Runtime/Lighting/Deferred.shader

```diff
@@ -128,17 +127,20 @@ Shader "Hidden/HDRP/Deferred"
                 // input.positionCS is SV_Position
                 float depth = LoadCameraDepth(input.positionCS.xy);
 
-                PositionInputs posInput = GetPositionInput(input.positionCS.xy, _ScreenSize.zw, depth, UNITY_MATRIX_I_VP, UNITY_MATRIX_V, uint2(input.positionCS.xy) / GetTileSize());
+                PositionInputs posInput = GetPositionInput(input.positionCS.xy, _ScreenSize.zw, depth, UNITY_MATRIX_I_VP, UNITY_MATRIX_V);
                 float3 V = GetWorldSpaceNormalizeViewDir(posInput.positionWS);
 
+                uint tile = ComputeTileIndex(posInput.positionSS);
```

Comment:
This should be move inside  LightLoop(), we must not change the prototype of LightLoop(), will break lot of stuff for users.

Reply:
I don't like the idea of tileCoord in the position input.
First, most shaders do not need it.
Secondly, you can clearly see that even our internal developers do not understand it, and put random stuff in there.

tileCoord is only useful for the light loop and should be its parameter. z-bin is debatable indeed.

This PR will introduce a lot of breaking changes, anyway.

Reply:
yes, you are right. TileCoord should be in input position.
And is something related to the ligthloop. But wondering how to abstract it. Because utimately what we want is to have lightloop being able itself from PositionInput (and maybe other stuff) to get the correct param for Zbinning or any algorithm we could come in the future.

>This PR will introduce a lot of breaking changes, anyway.
Still if we can still have users code compile it is a win. We will see at the end but maybe we will introduce a prototype with an ignore parameter just to ignore the compiler complain